src/appmixer/intelswift/core/HookUpdateTicket/HookUpdateTicket.js

```
@@ -0,0 +1,29 @@
+
+    async receive(context) {
+
+        const { platformType, syncId, ...meta } = context.messages.in.content;
+
+        // remaining to handles tenantId and projectId
+
+        let body = {};
+        if(!platformType) throw 'platformType not found';
+        if(!syncId) throw 'syncId not found';
+
+        if(platformType == 'jira') body.jiraId = syncId;
```

ESLint

[error] Expected space(s) after "if".

(keyword-spacing)